From 6e36aee7047aee8da7819597905464c15c6cbbfb Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 6 Nov 2018 15:24:57 +0100
Subject: [PATCH] [SQUSH][DNM] Merge tag 'android-9.0.0_r16' into lineage-16.0

commit cd725f092826c2bca5e9aef2ea7e308738e69151
Merge: 8da1fbb 94f7e57
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Fri Jul 27 03:15:09 2018 +0000

    Snap for 4916303 from 94f7e57d68b07f9fbfc7d15ad06c5467a4bc0afc to pi-qpr1-release

    Change-Id: Icd86ed5f0ecb1d46e2b072bbac6441f7ebed69e7

commit 94f7e57d68b07f9fbfc7d15ad06c5467a4bc0afc
Author: Bill Yi <byi@google.com>
Date:   Fri Jul 20 12:00:11 2018 -0700

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: I4ded0f91d9aa98ace6a47aae11f97d5da5555e94

Change-Id: Ifdf3dd1445d9fed636f48e8994d85b20c7cb1787
---
 res/values-sw/strings.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/res/values-sw/strings.xml b/res/values-sw/strings.xml
index 186a561..b6ee5f5 100644
--- a/res/values-sw/strings.xml
+++ b/res/values-sw/strings.xml
@@ -17,7 +17,7 @@
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="app_name" msgid="3551052199033657984">"Huduma Chaguo msingi ya Kuchapisha"</string>
+    <string name="app_name" msgid="3551052199033657984">"Huduma Chaguomsingi ya Kuchapisha"</string>
     <string name="printer_busy" msgid="8604311528104955859">"Inatumika"</string>
     <string name="printer_out_of_paper" msgid="4882186432807703877">"Karatasi zimeisha"</string>
     <string name="printer_out_of_ink" msgid="7361897651097675464">"Wino umeisha"</string>
-- 
2.17.1

